### PR TITLE
Added modal 'terms of service'

### PR DESCRIPTION
**Pull request by spavlishak**

### CODE REVIEW

File: app/views/shared/_footer.html.slim [hunk 1]
```diff
@@ -20,3 +20,5 @@ footer
         li: a target='_blank' href='https://peatio.tech/' About
         li: a target='_blank' href='https://github.com/rubykube/peatio/issues?state=open' Issues
         li: a target='_blank' href='https://github.com/rubykube/peatio/releases' Releases
+        li: a data-toggle="modal" data-target="#myModal" Terms of Service
```
Comment (yivo) [inline app/views/shared/_footer.html.slim]: @spavlishak I believe `#myModal` is not a good ID. Isnt it? Also we use single quotes. Please follow conventions.

Comment (yivo) [reply app/views/shared/_footer.html.slim]: @spavlishak Also you can simply the data attributes by writing ruby hash: `data={ toggle: 'modal', target: '#myModal' }`

File: app/views/shared/_terms.html.erb [hunk 2]
```diff
@@ -0,0 +1,16 @@
+<div class="modal fade" id="myModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel">
+  <div class="modal-dialog modal-lg" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+        <h4 class="modal-title" id="myModalLabel">Modal title</h4>
+      </div>
+      <div class="modal-body">
+        <iframe src="PEToS.pdf" width="100%" height="700px"></iframe>
```
Comment (yivo) [inline app/views/shared/_terms.html.erb]: @spavlishak You should put this file to app/assets/documents and use sprockets helpers to generate link to it.

File: app/views/shared/_terms.html.erb [hunk 3]
```diff
@@ -0,0 +1,16 @@
+<div class="modal fade" id="myModal" tabindex="-1" role="dialog" aria-labelledby="myModalLabel">
+  <div class="modal-dialog modal-lg" role="document">
+    <div class="modal-content">
+      <div class="modal-header">
+        <button type="button" class="close" data-dismiss="modal" aria-label="Close"><span aria-hidden="true">&times;</span></button>
+        <h4 class="modal-title" id="myModalLabel">Modal title</h4>
```
Comment (yivo) [inline app/views/shared/_terms.html.erb]: @spavlishak That's funny name. Please, don't copy-paste from BS examples. Write your own.

**Review by yivo — CHANGES_REQUESTED** [review]

**Comment by mod** [comment]
@spavlishak that's is the wrong repository.
you can't put the ToS on opensource.
you need to wipe this clean.

**Comment by mod** [comment]
please use peatio.tech where all the source code is public and no TOS can be added

**Pull request closed without merging**